core/store/src/migrations.rs

```diff
@@ -585,9 +626,9 @@ pub fn migrate_17_to_18(path: &String) {
 
 #[cfg(feature = "protocol_feature_rectify_inflation")]
 pub fn migrate_18_to_rectify_inflation(path: &String) {
```

Comment:
@bowenwang1996 I'm not really sure about this one. Whether it should use the old structures or new depends on which migration is applied first.

Reply:
This one will be applied first

Reply:
Ok, should be correct as written then.